docs/index.md

```diff
@@ -43,6 +43,12 @@ pip install imops[numba]  # additionally install Numba backend
 
 ::: imops.measure.label
 
+::: imops.measure.center_of_mass
+
 ::: imops.radon.radon
 
 ::: imops.radon.inverse_radon
+
+::: imops._numeric._sum
+
+::: imops._numeric._mul
```

Comment:
shouldn't include private functions and modules to public docs

Reply:
Same, see L39-43